xarray/core/datatree.py

```diff
@@ -1717,14 +1752,18 @@ def to_netcdf(
         kwargs :
             Additional keyword arguments to be passed to ``xarray.Dataset.to_netcdf``
 
+        Returns
+        -------
+            A bytes object with the byte content of the netCDF file, if filepath was None.
```

Comment:
Should the other possibilities be mentioned here, too?

Reply:
done!